model/Core/Classes/Process.md

```diff
@@ -0,0 +1,32 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# Process
```

Comment:
I think we agreed on renaming this to DefinedProcess or ProcessPlan or something...